### ISSUE TITLE
Ionic 5 ion-select does not show the selected values using Angular 10 

### ISSUE THREAD

**Issue by edocbuhtig** [ionic-team/ionic-framework#22705]
# Bug Report

**Ionic version:**

[ ] **4.x**
[x] **5.x**

**Current behavior:**
When you dynamically populate values to ion-select using *ngFor the ion-select is blank, the selected patchValue appears only after clicking/tapping on ion-select.

Dirty fix is that you use:  
```
setTimeout(() => {
   this.form.patchValue({ ... });
}, 500);
```

**Expected behavior:**
ion-select must show the last selected values stored in the backend, when populating ion-select using *ngFor on the client reactive form using patchValue, without user interaction.

**Related code:**
`ion-select` Dynamic Data List 
```
[ 
  { 
	"ctitle": "Category 1", 
	"dt": "Wed Dec 16 2020 20:31:05" 
  },
  { 
	"ctitle": "Category 2", 
	"dt": "Wed Dec 16 2020 20:31:05" 
  },
  { 
	"ctitle": "Category 3", 
	"dt": "Wed Dec 16 2020 20:31:05" 
  }  
]
```

Reactive Form
```
<ion-select [formControl]="editForm.controls['cat']" okText="Select" cancelText="Cancel">
  <ion-select-option *ngFor="let catfeeds of (categorylist | async)" value="{{catfeeds.ctitle}}">{{catfeeds.ctitle}}</ion-select-option>
</ion-select>
```

Data to patch to ion-select using `editForm.controls['cat'].patchValue({})`
```
"ctitle": "Category 2"
```

**Other information:**

Related discussion still open for a solution and hoping for a FIX in next release of Ionic v5.x.x
https://forum.ionicframework.com/t/ionic-5-angular-ion-select-does-not-show-the-selected-values/186633/31

**Ionic info:**

```
Ionic:

   Ionic CLI                     : 6.12.3 
   Ionic Framework               : @ionic/angular 5.5.2
   @angular-devkit/build-angular : 0.1000.8
   @angular-devkit/schematics    : 10.0.8
   @angular/cli                  : 10.0.8
   @ionic/angular-toolkit        : 2.3.3

Capacitor:

   Capacitor CLI   : 2.4.2
   @capacitor/core : 2.4.2

Utility:

   cordova-res                          : 0.15.2
   native-run (update available: 1.3.0) : 0.2.9

System:

   NodeJS : v14.15.2 
   npm    : 6.14.9
   OS     : macOS Catalina
```

**Comment by keo9ren** [comment]
Possibly same as this one #19324

**Comment by liamdebeasi** [comment]
Thanks for the issue. I am going to close this as a duplicate of https://github.com/ionic-team/ionic-framework/issues/19324.
